### DISCUSSION TITLE
select and display data from table

### DISCUSSION THREAD

**Discussion by summX** [Budibase/budibase#13270] [Help] [answered]
HI. It´s my first day with budibase and I´m not a coder. I´m looking for a way to achieve something in my timekeeping project: I generate a table of students who have faults in their timekeeping by querying the table. The results are displayed as intended: a table with names of students. But I can´t figure out how to pull up a single student from that list and get a listing of all the dates with faulty entries to correct them manually. I´ve been told to use state, but I´m simply not getting the hang of it. Any help is appreciated.

**Comment by summX** [comment, marked as answer]
I´ve solved it! can be deleted

**Reply by ConorWebb96** [reply]
I'm glad you figured this out, can I ask were you found the answer to this issue?

Thanks in advance for this information.

**Reply by summX** [reply]
Hi Conor. I inspected examples from the library and kept an eye on filter settings and handlebar code. I still don´t fully understand it as I´m not a "code" person at all ;)
